karspexet/templates/base.html

```diff
@@ -0,0 +1,91 @@
+{% load menu_tags cms_tags sekizai_tags staticfiles%}
+<html>
+    <head>
+        <title>{% page_attribute "page_title" %}</title>
+        <link rel="shortcut icon" href="{% static 'favicon.ico' %}">
+        {% render_block "css" %}
+    </head>
+    <body>
+        {% cms_toolbar %}
+        <div id="navbar">
+            <div class="horizontal titleContainer">
+              <h1><span class="g">KÅR</span>S<span class="g">P</span>EX<span class="g">ET</span></h1>
+            </div>
+            
+            <div class="horizontal menuContainer">
+              <ul class="menu">
+                {% show_menu 0 100 0 1 %}
+              </ul>
+            </div>
+        </div>
+        {% addtoblock "css" %}
```

Comment:
At least in the same template file. If you inherit the layout and want to add a page specific script, then this is the way to go. But here? Not so much, IMO.